articles/ai-services/document-intelligence/faq.yml

```diff
@@ -134,7 +134,13 @@ sections:
         answer: |
          **Yes.**
 
-         If your Document Intelligence resource is configured with a firewall or virtual network, you need to add the dedicated IP address 20.3.165.95 to the firewall allowlist for your Document Intelligence resource.
+         If your Document Intelligence resource is configured with a firewall or virtual network, you need to add the dedicated IP address 20.3.165.95 to the firewall allowlist for your Document Intelligence resource.  Please note that some functions in custom projects (e.g. auto-label, project management and human in the loop) won't work if the public network access is disabled.
+
+      - question: |
+         When I upload a file in Document Intelligence Studio by "Fetch from URL" function, can I use a URL from my blob storage?
+
+        answer: |
+         Yes if it's a URL with sas token and your storage account is accessible from public network.  Fetching from URL does not work for storage account with key access disabled or behind firewall/VNet.
```

Comment:
```suggestion
      - question: |
         When I upload a file in Document Intelligence Studio by "Fetch from URL" function, can I use a URL from my blob storage?

        answer: |
         **Yes.** 
         
         If your Azure blob storage URL includes a SAS token and is accessible from public networks.  You cannot use the **Fetch** function for storage accounts where the key access is disabled or behind a firewall/VNet.
```

Reply:
@esther-tsai, please accept the above changes and I can sign off on this PR.

Reply:
The change looks good to me.  Thanks for the suggestion!